src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -96,6 +99,39 @@ class ComService(
       communityApiClient.getTeamsCodesForUser(body.staffIdentifier),
       probationSearchApiSortBy,
     )
-    return entityProbationSearchResult.transformToModelProbationResult()
+
+    val enrichedProbationSearchResult = entityProbationSearchResult.mapNotNull {
+      val licences =
+        licenceRepository.findAllByCrnAndStatusCodeIn(it.identifiers.crn, LicenceStatus.searchRelevantLicenceStatus())
+
+      // If an empty list has been returned, there are no relevant licences relating to search for the offender
+      if (licences.isEmpty()) {
+        null
+      } else {
+        val nonActiveLicenceStatuses = LicenceStatus.searchRelevantLicenceStatus().minusElement(LicenceStatus.ACTIVE)
```

Comment:
think [minusElement](https://github.com/JetBrains/kotlin/blob/7a7d392b3470b38d42f80c896b7270678d0f95c3/libraries/stdlib/common/src/generated/_Collections.kt#L3178) is an alias for minus?

so can presumably call it as the `-` operator?

Reply:
Done